halyard-web/pkg_scripts/postInstall.sh

```diff
@@ -11,7 +11,11 @@ if [ -f "/opt/spinnaker/config/halyard-user" ]; then
 fi
 
 if [ -z "$HAL_USER" ]; then
-  HAL_USER="ubuntu"
+  getent passwd spinnaker > /dev/null
```

Comment:
Great. 
It seems various scripts in the repo all support supplying a user for Halyard to run as.
`spinnaker` is a sensible default, matching other services like clouddriver, etc.

Do we need to also write the user into `/opt/spinnaker/config/halyard-user` ?
there look to be a few "update" related scripts in the repo that might depend on this as well?

Thanks again for tackling this. Big help.

Reply:
I've now updated my changes to include this.